ppanggolin/cluster/cluster.py

```diff
@@ -447,11 +498,19 @@ def read_clustering(pangenome: Pangenome, families_tsv_path: Path, infer_singlet
     :param disable_bar: Allow to disable progress bar
     """
     check_pangenome_former_clustering(pangenome, force)
-    check_pangenome_info(pangenome, need_annotations=True, need_gene_sequences=True, disable_bar=disable_bar)
+
+    if pangenome.status["geneSequences"] == "No":
```

Comment:
since we're in a "read external cluster' case, shouldn't we just not load gene sequences even if there are some?

Reply:
This is used to add protein sequences of representative genes at the end. 
It's used when context and fasta commands are apply to the pangenome I believe.
Since pangenomes made with internal clustering already contain these sequences, I guess it is better to also have them in pangenomes generated with external clustering when it is possible to ensures that both types of pangenomes behave consistently...